tests/qos/qos_sai_base.py

```diff
@@ -1664,12 +1665,17 @@ def populateArpEntries(
     @pytest.fixture(scope='class', autouse=True)
     def dut_disable_ipv6(self, duthosts, get_src_dst_asic_and_duts, tbinfo, lower_tor_host): # noqa F811
         for duthost in get_src_dst_asic_and_duts['all_duts']:
+            docker0_ipv6_addr = \
+            duthost.shell("sudo ip -6  addr show dev docker0 | grep global" + " | awk '{print $2}'")["stdout_lines"][0]
             duthost.shell("sysctl -w net.ipv6.conf.all.disable_ipv6=1")
 
         yield
 
         for duthost in get_src_dst_asic_and_duts['all_duts']:
             duthost.shell("sysctl -w net.ipv6.conf.all.disable_ipv6=0")
+            logger.info("Adding docker0's IPv6 address since it was removed when disabing IPv6")
+            duthost.shell("ip -6 addr add {} dev docker0".format(docker0_ipv6_addr))
+            config_reload(duthost, config_source='config_db')
```

Comment:
add options to wait for interfaces to be up ? 
config_reload(duthost, config_source='config_db', safe_reload=True, check_intf_up_ports=True)